app/components/windows/SourcesShowcase.vue

```diff
@@ -8,17 +8,15 @@
     class="add-source">
     <!-- Standard sources -->
     <add-source-info
-      v-for="source in availableSources"
-      :key="source.value"
-      v-if="inspectedSource === source.value"
-      :name="sourceData(source.value).name"
-      :description="sourceData(source.value).description"
-      :showSupport="!!sourceData(source.value).supportList"
+      v-if="inspectedSourceDefinition"
```

Comment:
Aren't we missing `key` here? Disregard if not needed.

EDIT: nevermind, this ain't a container.